liquibase-cli/pom.xml

```diff
@@ -24,6 +24,7 @@
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-standard</artifactId>
             <version>${project.version}</version>
+            <scope>compile</scope>
```

Comment:
I assume the intention was to mark this dependency as compile-only, but `compile` is the default scope (per [Maven documentation](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#:~:text=compile%0AThis%20is%20the%20default%20scope%2C%20used%20if%20none%20is%20specified.)), i.e., this diff has led to no effective change.